root/user/UserCollections.js

```diff
@@ -0,0 +1,199 @@
+  ) : (
+    collaborators.length
+  );
+}
+
+function formatPrivacy(isPublic: boolean) {
+  return isPublic ? l('Public') : l('Private');
+}
+
+function collaborativePrivacy(
+  collection: CollectionT,
+  currentUser: EditorT | null,
+) {
+  return !!currentUser && !!collection.editor &&
+    collection.editor.id === currentUser.id ? (
+      formatPrivacy(collection.public) + ' ' + l('(your collection)')
+    ) : (
+      formatPrivacy(collection.public)
+    );
+}
+
+const CollectionsEntitySection = ({
+  $c,
+  isCollaborative,
+  collections,
+  type,
+  user,
+}) => {
+  const viewingOwnProfile = !!$c.user && $c.user.id === user.id;
+  const nameColumn = defineNameColumn(N_l('Collection'));
+  const sizeColumn = {
+    Header: () => formatPluralEntityTypeName(type),
+    accessor: 'entity_count',
+    id: 'size',
+  };
+  const collaboratorsColumn = {
+    Cell: ({cell: {value}}) => formatCollaboratorNumber(value, $c.user),
+    Header: N_l('Collaborators'),
+    accessor: 'collaborators',
+    id: 'collaborators',
+  };
+  const privacyColumn = {
+    Cell: ({row: {original}}) => collaborativePrivacy(original, $c.user),
+    Header: N_l('Privacy'),
+    accessor: 'public',
+    id: 'privacy',
+  };
+  const actionsColumn = defineActionsColumn([
+    [l('Edit'), '/edit'],
+    [l('Remove'), '/delete'],
+  ]);
+  const columns = React.useMemo(
+    () => [
+      nameColumn,
+      typeColumn,
+      sizeColumn,
+      collaboratorsColumn,
+      ...viewingOwnProfile ? [subscriptionColumn] : [],
+      ...viewingOwnProfile || isCollaborative ? [privacyColumn] : [],
+      ...viewingOwnProfile && !isCollaborative ? [actionsColumn] : [],
+    ],
+    [],
+  );
+
+  return (
+    <>
+      <h3>{unwrapNl(collectionsListTitle[type])}</h3>
+      <Table columns={columns} data={collections} />
+    </>
+  );
+};
+
+const UserCollections = withCatalystContext(({
+  $c,
+  ownCollections,
+  collaborativeCollections,
+  user,
+}: Props) => {
+  const viewingOwnProfile = !!$c.user && $c.user.id === user.id;
+
+  return (
+    <UserAccountLayout
+      entity={user}
+      page="collections"
+      title={l('Collections')}
+    >
+      <h2>{l('My collections')}</h2>
+      {Object.keys(ownCollections).length > 0 ? (
+        Object.keys(ownCollections).sort().map(type => (
+          <CollectionsEntitySection
+            $c={$c}
+            collections={ownCollections[type]}
+            isCollaborative={false}
+            key={type}
+            type={type}
+            user={user}
+          />
+        ))
+      ) : (
+        <p>
+          {viewingOwnProfile ? (
+            l('You have no collections.')
+          ) : (
+            exp.l('{user} has no public collections.', {user: user.name})
+          )}
+        </p>
+      )}
+      <h2>{l('Collaborative collections')}</h2>
+      {Object.keys(collaborativeCollections).length > 0 ? (
+        Object.keys(collaborativeCollections).sort().map(type => (
```

Comment:
Same as above

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/1246/commits/6ee5d3d424b1dda8aecefe6be1b9f7d2ac4092e1#diff-3f513fa6fa1fed40110c8c20d602c01fR139